src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileLength/neversuppressed/case1/defaultContextConfig.xml

```diff
@@ -8,6 +8,6 @@
   </module>
   <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
     <property name="file" value="${tp}/defaultContext.patch" />
-    <property name="strategy" value="patchedline" />
```

Comment:
why do you change strategy for these checks?
I think strategy should stay and you just add neverSuppressedChecks property.

Reply:
done